index.html

```diff
@@ -0,0 +1,18 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="/vite.svg" />
```

Comment:
I would link to an external file and remove vit.svg file. https://github.com/CartoDB/viz-doc/blob/master/guides/public-app/index.html#L5